src/test/rustdoc-js/generics-multi-trait.js

```diff
@@ -0,0 +1,30 @@
+const QUERY = [
```

Comment:
I think it'd be better to have an "exact check". To do so, please add `// exact-check` at the beginning of the file.

Reply:
You're right.